estela_cli/init.py

```diff
@@ -21,10 +22,10 @@
 )
 
 
-SHORT_HELP = "Initialize estela project for existing scrapy project"
+SHORT_HELP = "Initialize estela project for existing crawling project"
```

Comment:
@joaquingx I think web scraping is probably a more standard term here (regardless of technicalities). cc: @mgonnav

Reply:
I agree, done.